database/redis/metric.go

```diff
@@ -264,7 +268,13 @@ func (connector *DbConnector) needRemoveMetrics(metric string) bool {
 }
 
 var patternsListKey = "moira-pattern-list"
-var metricEventKey = "{channel}:metric-event"
+
```

Comment:
As a variant - use parameter like `event_sharded` (boolean). Also number of shards can be used (but need to sync beetween and notifiers).

Reply:
Thanks for your suggestion, but I think it's premature.